board/aarch64/bananapi-bpi-r3/dts/mediatek/mt7986a-bananapi-bpi-r3-emmc.dtsi

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+/*
+ * Copyright (C) 2021 MediaTek Inc.
+ * Author: Sam.Shih <sam.shih@mediatek.com>
+ */
+
+/*
+This is copied from linux where it is an overlay, unfortunatly it is not
+possible to use dtbo in sysboot unless present in syslinux.conf
```

Comment:
Not sure if it makes sense or not, but the standard scripts do check for a board-specific `syslinux.conf` before falling back to the generic one:
https://github.com/kernelkit/infix/blob/0328700afb3956e75cb9a4c4f2c592905d129b1b/board/common/uboot/scripts/ixbootslot.sh#L37-L41